src/main/java/com/spotify/github/v3/clients/JwtTokenIssuer.java

```diff
@@ -30,37 +30,55 @@
 import java.security.spec.KeySpec;
 import java.security.spec.PKCS8EncodedKeySpec;
 import java.util.Date;
+import java.util.function.Supplier;
 
 /** The helper Jwt token issuer. */
 public class JwtTokenIssuer {
 
   private static final SignatureAlgorithm SIGNATURE_ALGORITHM = SignatureAlgorithm.RS256;
-  private static final long TOKEN_TTL = 600000;
+  private static final long TOKEN_TTL = 600_000;
+  private static final long TOKEN_ISSUED = 60_000;
```

Comment:
Fun Fact: Java totally allows _ in numbers for readability;

Reply:
[linky](https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html)